gobcore/message_broker/messagedriven_service.py

```diff
@@ -72,7 +48,7 @@ def _on_message(connection, service, msg):
         stacktrace = traceback.format_exc(limit=-5)
         print("Message processing has failed, further processing stopped", stacktrace)
         # Log the error and a short error description
-        log_error(f"Message processing has failed, further processing stopped", msg, err)
+        logger.gob_core_error(f"Message processing has failed, further processing stopped", msg, err)
```

Comment:
The logger is not configured.
It should be configured with the message and name "Core".
Then logger.error should be called here.

Reply:
The message might not be set correctly within this context, we could validate the message and try to configure the logger for CORE with the message.
And fallback to current implementation if that fails.